Resources/Prototypes/_NF/Recipes/Reactions/drinks.yml

```
@@ -8,3 +8,29 @@
       amount: 1
   products:
     Coffee: 4
+
+- type: reaction
+  id: PumpkinSpiceLatte
+  reactants: # Amounts might be annoying, but it's easier to halve for easy brain math.
+    Coffee:
+      amount: 4
+    PumpkinFlesh:
+      amount: 2
+    Sugar:
+      amount: 2
+    Milk:
+      amount: 1
+    Blackpepper:
+      amount: 1
+  products:
+    PumpkinSpiceLatte: 10
+
+- type: reaction
+  id: GinAndSonic
+  reactants: # Amounts might be annoying, but it's easier to halve for easy brain math.
+    GinTonic:
+      amount: 5
+    Desoxyephedrine:
+      amount: 5
+  products:
+    PumpkinSpiceLatte: 10
```

Ratios don't need to add up to 10 - if you can, keep your ratios as small as possible.
```suggestion
  reactants:
    GinTonic:
      amount: 1
    Desoxyephedrine:
      amount: 1
  products:
    GinAndSonic: 2
```